app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+  # before_action :configure_account_update_params, only: [:update]
+
+  # GET /resource/sign_up
+  def new
+    super do
+      resource.user_profile = UserProfile.new
+    end
+  end
+
+  # POST /resource
+  def create
+    super
+  end
+
+  # GET /resource/edit
+  # def edit
+  #   super
+  # end
+
+  # PUT /resource
+  # def update
+  #   super
+  # end
+
+  # DELETE /resource
+  # def destroy
+  #   super
+  # end
+
+  # GET /resource/cancel
+  # Forces the session data which is usually expired after sign
+  # in to be expired now. This is useful if the user wants to
+  # cancel oauth signing in/up in the middle of the process,
+  # removing all OAuth session data.
+  # def cancel
+  #   super
+  # end
+
+  protected
+
+  # If you have extra params to permit, append them to the sanitizer.
+  def configure_sign_up_params
+    devise_parameter_sanitizer.permit(:sign_up, keys: [
+                                        user_profile_attributes: %i[
```

Comment:
What happens here? %i?

Reply:
@marcfreiheit

Reply:
This is a array of symbols

https://stackoverflow.com/questions/39303737/why-do-you-use-w-in-rails

Reply:
Hm ok. Do you use that for a reason? Or just style?